src/components/AppBar/AppBar.js

```diff
@@ -291,15 +297,10 @@ export default function AppBarComponent({
                                         Login
```

Comment:
Change to 
`href={"/login"}`

since you changed the directory name from `signin` to `login`